v1/proto/service/gribi.proto

```diff
@@ -460,9 +464,11 @@ message FlushRequest {
     //     the Flush is accepted.
     //   - if the election ID is less than the previous highest known ID, the
     //     Flush is rejected and an error with reason NOT_PRIMARY is returned.
+    //     The canonical error code used should be FAILED_PRECONDITION.
     //
     // If the server is currently operating in ALL_PRIMARY mode the server
-    // should return an error specifying ELECTION_ID_IN_ALL_PRIMARY reason.
+    // should return an error specifying ELECTION_ID_IN_ALL_PRIMARY reason. The
+    // status (canonical error code) should be FAILED_PRECONDITION.
     Uint128 id = 1;
```

Comment:
We also need a `Reason` error enum for the case where election_id is 0 (since 0 is an invalid election_id).
0 is disallowed in Modify.

Reply:
Thanks, added  `INVALID_ELECTION_ID`.